salt/_pillar/metalk8s.py

```diff
@@ -120,7 +120,6 @@ def _load_iso_path(config_data):
 
     return res
 
-
```

Comment:
`salt/_pillar/metalk8s.py:123:1: E302 expected 2 blank lines, found 1`

Reply:
We should have one but it's not only about enable pylint/flake8/whatever because salt have some specific variables.
Just lets try to follow python good practice even if we have not automatic linting